### ISSUE TITLE
resources not converted to globs

### ISSUE THREAD

**Issue by joprice** [pinterest/PodToBUILD#7]
Resources don't seem to get converted into glob patterns. I get errors like this:

`invalid label 'Source/**/*.{js,css}' in element 0 of attribute 'resources' in 'objc_library' rule: invalid target name 'Source/**/*.{js,css}': target names may not contain '*'`

**Comment by rahul-malik** [comment]
+ @jerrymarino 

@joprice  - Can you link the portion of the podspec (or the entire thing) that is causing this issue?

**Comment by joprice** [comment]
```
  s.resources = [
    'Source/**/*.{js,css}',
    'Source/Resources/*.xcassets',
    'Source/Resources/Fonts/**/*.{otf,ttf}'
  ]
```

From https://github.com/FolioReader/FolioReaderKit/blob/master/FolioReaderKit.podspec

**Comment by jerrymarino** [comment]
Hey @joprice - thanks for using `PodToBUILD` and for the issue!

I've had a min to look into this and test, I've gotten the `resources` bit of this fixed, which looks like we didn't correctly parse them. https://github.com/pinterest/PodToBUILD/pull/9/commits/713def8a78d038fc6e9059fe7bd171c671ebe612#diff-ad62a88829aece0679e97251dd612946

We'll need to look at adding swift support for this pod!

**Comment by jerrymarino** [comment]
I'm currently heading offline for vacation for a few days and will update once I'm back.

**Comment by joprice** [comment]
No problem. Thanks for looking into it. I'd be wary of adding this pod as an integration test though, since it depends on Realm, which has long build times and results in huge artifacts. I forked the project and replaced the Realm integration with a protocol on this branch: https://github.com/joprice/FolioReaderKit/tree/noRealm. Mind you I'm new to Swift, iOS, and Bazel, so not sure if there's a better solution.

**Comment by jerrymarino** [comment]
@joprice welcome to the party! We currently "integration test" output at the BUILD file compiler level to make sure our output is correct and idempotent. We don't validate if the `examples` are building in this program right now, but this is something I'd love to add in the future: A CI that continuously builds the latest version of the top 100 pods with Bazel.

The core issue you reported should be fixed by https://github.com/pinterest/PodToBUILD/pull/9.

I imagine the overall problem of building `FoloioKit` will persist until we get support for swift. The core mechanics of how swift will work are similar to how other languages will work, I'll write up an issue for that.

**Comment by joprice** [comment]
I'm not sure that did the trick. I get an error like this: 

```
//Vendor/FolioReaderKit:FolioReaderKit: invalid label 'Source/**/*.css' in element 0 of attribute 'resources' in 'objc_library' rule: invalid target name 'Source/**/*.css': target names may not contain '*'
```

I think the generated resources are supposed to be wrapped in a call to `glob()`?

**Comment by jerrymarino** [comment]
@joprice you are right. I’ll push a fix for that this afternoon - we just need wrap the array in a GlobNode.

BTW - Overall it is kind of hard to fully test without swift_library fully working. If all goes well I’m going to take a stab at this later today  https://github.com/pinterest/PodToBUILD/issues/11
